### PR TITLE
Fix night theme logo in docs

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by s-santillan** [comment]
What I think you need to change is the asset used here: https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep.svg

It's in the semgrep repo.

cc @pabloest

**Review by s-santillan — COMMENTED** [review]
Alternatively, you can upload to static folder, same as our other photos.

**Comment by adamkvitek** [comment]
@s-santillan changing the [asset](https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep.svg) these assets need to be two for dark and light mode. Using the dark asset instead of the current would not make the font useful in another mode.

**Comment by adamkvitek** [comment]
@s-santillan image loading from a static file done! :)

File: docs/index.md [hunk 1]
```diff
@@ -10,11 +10,17 @@ description: >-
...
+import ThemedImage from '@theme/ThemedImage'
 
 <br />
 <p align="center">
   <a href="https://semgrep.dev">
-    <img src="https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep.svg" height="100" alt="Semgrep logo" />
+    <ThemedImage
+      alt="Semgrep themed logo"
+      sources={{
+        light: ('https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep.svg'),
```
Comment (pabloest) [inline docs/index.md:21]: @adamkvitek Can you make this file a local asset as well?

Comment (adamkvitek) [reply docs/index.md]: @pabloest done in: https://github.com/returntocorp/semgrep-docs/pull/881/commits/a1056ba72f2d787aa686ad2797ba061aff796c92 Please check :-)

**Review by pabloest — APPROVED** [review]

**Pull request merged**